ngx-tools/utilities/src/number-abbreviation/number-abbreviation.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { numberAbbreviation } from './number-abbreviation';
+
+
+describe(`numberAbbreviation`, function() {
+
+  test(`should parse to the lowest level specified`, () => {
+    const input1 = 1234;
+    const input2 = 1234567;
+    const input3 = 1234567890;
+    const input4 = 1234567890123;
+    const input5 = 1200;
+    const input6 = 450;
+    const input7 = 123456;
+    expect(numberAbbreviation(input1, 1)).toEqual('1.2K');
+    expect(numberAbbreviation(input2, 1)).toEqual('1.2M');
+    expect(numberAbbreviation(input3, 1)).toEqual('1.2B');
+    expect(numberAbbreviation(input4, 2)).toEqual('1.23T');
+    expect(numberAbbreviation(input5, 3)).toEqual('1.200K');
+    expect(numberAbbreviation(input6, 1)).toEqual('450');
+    expect(numberAbbreviation(input7, 2)).toEqual('123.46K');
```

Comment:
Also just realized, we aren't testing the default decimal either. ie `nA(input).toEqual..`